### ISSUE TITLE
Excessive log file generation (using gigabytes of disk space)

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#1460]
Causes hosts to eventually crash by filling up disk space.

```
$ sudo du -cha --max-depth=1 /home/ubuntu/.cache/nucypher/log/ | grep -E "M|G"
273M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_6
259M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_8
115M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_10
256M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_10
264M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_7
64M     /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_4
122M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_6
86M     /home/ubuntu/.cache/nucypher/log/nucypher.log
118M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_7
286M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_5
116M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_9
112M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_11
116M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_8
251M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_11
258M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json.2019_11_9
127M    /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_5
29M     /home/ubuntu/.cache/nucypher/log/nucypher.log.2019_11_4
194M    /home/ubuntu/.cache/nucypher/log/nucypher.log.json
3.0G    /home/ubuntu/.cache/nucypher/log/
3.0G    total

```

**Comment by michwill** [comment]
About 40MB per file for me. So not as bad, but indeed, that's a bit too much

**Comment by derekpierre** [comment]
Are the logs essentially filled with learning loop entries? We can probably tame the logs for the monitor.

**Comment by tuxxy** [comment]
https://linux.die.net/man/8/logrotate

**Comment by derekpierre** [comment]
Related to #1712 .

**Comment by KPrasch** [comment]
Fixed in #1460
